content/download/index.html.haml

```diff
@@ -92,6 +92,10 @@ title: Jenkins download and deployment
               %div{:style => 'font-size: x-small;overflow-wrap: break-word; word-wrap: break-word;'}
                 SHA-256:
                 = URI.open('https://repo.jenkins-ci.org/releases/org/jenkins-ci/main/jenkins-war/' + site.jenkins.stable + '/jenkins-war-' + site.jenkins.stable + '.war.sha256').read
+                %span{class: 'sha-256', data: {clipboard_text: URI.open('https://repo.jenkins-ci.org/releases/org/jenkins-ci/main/jenkins-war/' + site.jenkins.stable + '/jenkins-war-' + site.jenkins.stable + '.war.sha256').read}} 
```

Comment:
Instead of calling the same logic again, the URL should be assigned to a variable to avoid redundancy.
Same like below.